packages/block-editor/src/components/iframe/index.js

```diff
@@ -141,13 +141,6 @@ function Iframe( {
 		function onLoad() {
 			const { contentDocument, ownerDocument } = node;
 			const { documentElement } = contentDocument;
-			// Get any CSS classes the iframe document body may initially have
-			// to re-apply them later together with the ones of the main document
-			// body. This is necessary for some CSS classes for example the
-			// `is-dark-theme` class added by useDarkThemeBodyClassName.
-			const initialIframeBodyClasses = Array.from(
-				contentDocument.body.classList
-			);
```

Comment:
Why not just revert this part?

Reply:
because `.concat( initialIframeBodyClasses )` is not relevant without the above, and `is-dark-theme` check at this point may also not be relevant, but it can be added back if needed in a follow up PR that gets `is-dark-theme` working in the post editor, otherwise it seemed like there was potential for it to be left there as dead code.

Reply:
If I took your comment to "So we should remove all this logic introduced last week" too literally 😄  I can put up a PR to add back in the `|| name === 'is-dark-theme' tomorrow if you think it is worth it.

Reply:
18.1 has already been released with the revert. 18.2 will be cut in 2 weeks if it can wait. Thanks!

Reply:
yep, I don't believe it will be an urgent change if we do add it back, 18.2 will be fine I think.